.github/workflows/ci.yml

```diff
@@ -17,23 +17,22 @@ on:
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
 
-env:
-  # Run only on PRs from the main repo, on scheduled, or triggered runs.
-  RUN_ON_PR_AND_SCHEDULE_AND_TRIGGER: github.event.pull_request.head.repo.full_name == github.repository || github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
```

Comment:
This was not getting correctly evaluated. Not for the existing `docs-snippets` steps as well